packages/react-components/src/AzeroId.tsx

```diff
@@ -132,17 +126,36 @@ const WrappedAzeroId = ({ address, className, isRegisterLinkShown = true }: Wrap
   return (
     <AzeroId
       address={address}
-      api={api}
       chainId={chainId}
       className={className}
       isRegisterLinkShown={isRegisterLinkShown}
     />
   );
 };
 
+export const AzeroIdDomain = ({ className, domain, isCopyShown = false, isLogoShown = true }: {className?: string, domain: string, isLogoShown?: boolean, isCopyShown?: boolean}) => {
+  const theme = useTheme();
+
+  return (
+    <DomainContainer className={className}>
+      {isLogoShown && (
+        <Logo
+          src={theme.theme === 'dark' ? externalAzeroIdLogoPrimarySVG : externalAzeroIdLogoBlackSVG}
+        />
+      )}
+      <span>
+        {domain.split(/(?=\.)/).map((domainPart, index) => <span key={index}>{domainPart}<wbr /></span>)}
+        {isCopyShown && <SmallIcon icon='copy' />}
+      </span>
+    </DomainContainer>
+  );
+};
+
+export const AZERO_ID_ROW_HEIGHT = '18px';
```

Comment:
https://youtu.be/ri83v76PXdE?si=xHEc3UIoX1heh-Nn&t=85

Once I heard it, I can't unhear it 😆

Reply:
hahaha :heart: